test/fixtures/section-heading-first-fail.html

```diff
@@ -0,0 +1,3 @@
+<section>
+    <p>Hello, world.</p>
+</section>
```

Comment:
It checks for the various levels of headings, H1 through H6, each of which are valid. Since the rule checks if the first child is H1-H6, we could only have one piece of markup in the "pass" file so long as it was one H1-H6.